pynvim/plugin/host.py

```diff
@@ -23,6 +22,27 @@
 host_method_spec = {"poll": {}, "specs": {"nargs": 1}, "shutdown": {}}
 
 
+def handle_import(directory, name):
+    """Import a python file given a known location.
+
+    Currently works on both python2 or 3.
+    """
+    try:  # Python3
+        from importlib.util import module_from_spec, spec_from_file_location
+    except ImportError:  # Python2.7
+        import imp
+        from pynvim.compat import find_module
+        file, pathname, descr = find_module(name, [directory])
+        module = imp.load_module(name, file, pathname, descr)
+        return module
+    else:
+        spec = spec_from_file_location(name, location=directory)
```

Comment:
I don't think this is correct -- this seems to be always returning `None`, causing `ImportError`. `
```
spec_from_file_location("foobar", "/path/to/foobar.py") == ModuleSpec(name='foobar', ...)
spec_from_file_location("foobar", "/path/to/") == None
```